README.hi.md

```
@@ -0,0 +1,441 @@
+node gemini-eval.mjs "We are looking for a Senior AI Engineer..."
+node gemini-eval.mjs --file ./jds/my-job.txt
+npm run gemini:eval -- "JD text here"
+```
+
+> **Free tier:** दोनों options billing के बिना काम करते हैं। Native CLI Google OAuth use करता है; API script `gemini-2.5-flash` use करता है (15 RPM, 1M tokens/day free)।
+
+## Usage
+
+Career-ops एक shared command router use करता है। CLIs में जो slash commands register करते हैं, यह इस तरह दिखता है:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)